src/oracle/PCVOracle.sol

```diff
@@ -108,6 +135,34 @@ contract PCVOracle is IPCVOracle, CoreRefV2 {
         return (oracleValue * venueBalance) / Constants.ETH_GRANULARITY;
     }
 
+    /// @notice returns decimal normalized version of a given venues stale USD balance
+    /// does not account for unearned yield
+    function getVenueStaleBalance(address venue) public view returns (uint256) {
+        uint256 venueBalance = venueRecord[venue]
+            .lastRecordedBalance
+            .toUint256();
+
+        return venueBalance;
+    }
```

Comment:
This feels like a duplication of `lastRecordedBalance(address venue)` with only an added cast to uint256, I'd remove

Reply:
removed